### ISSUE TITLE
Zuora configurator

### ISSUE THREAD

**Issue by rndstr** [weaveworks/service#1444]
_From @jml on September 7, 2017 14:26_

We have to set a hojillion options on Zuora to make this work. We currently do this manually. This sucks, and is not very GitOps.

I don't know if the Zuora API supports this, but we should have:
- all our config in a parseable text file(s) in a git repo
- a tool for applying this to Zuora
- a tool for getting our current Zuora config in a similar format
- something to see the diff between current Zuora config and intended Zuora config
- an alert for when the configs differ

Basically, what we do for kubediff, but for Zuora.

_Copied from original issue: weaveworks/billing#201_
